test_cli.py

```diff
@@ -109,23 +110,28 @@ def get_info(uuid, key):
     return _run_command([cl, 'info', '-f', key, uuid])
 
 
-def wait_until_running(uuid, timeout_seconds=100):
+def wait_until_state(uuid, expected_state, timeout_seconds=100):
     start_time = time.time()
     while True:
-        if time.time() - start_time > 100:
+        if time.time() - start_time > timeout_seconds:
```

Comment:
Can you add a docstring with the logic?  It's not obvious what role FINAL_STATES plays.

Reply:
I added a docstring and made the logic clearer by explicitly raising an `AssertionError` if the bundle is in a final state that is not the expected state.